.github/workflows/examples_js.yml

```diff
@@ -79,12 +79,12 @@ jobs:
         if: ${{ failure() && github.ref == 'refs/heads/master' && !contains(github.event.head_commit.message, 'release') && !contains(github.event.head_commit.message, 'Release') && !inputs.dont_notify }}
         uses: rtCamp/action-slack-notify@v2
         env:
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_TITLE: "JS Example Tests Failed"
+          SLACK_WEBHOOK: ${{ secrets.SLACK_TECH_WEBHOOK }}
+          SLACK_TITLE: "Example Tests Failed"
           SLACK_MSG_AUTHOR: ${{ inputs.author || github.actor }}
-          SLACK_MESSAGE: "<@team> ${{ inputs.commit_message || github.event.head_commit.message }}"
+          SLACK_MESSAGE: "<@Apoorv> <@kaavee> ${{ inputs.commit_message || github.event.head_commit.message }}"
```

Comment:
Hardcoding specific user mentions (`@Apoorv` and `@kaavee`) in the workflow file is not a maintainable practice. Consider:
1. Using a team mention instead (`@team` as before)
2. Or using a configurable environment variable/secret that can store the list of users to notify
This would make it easier to update notification targets without changing the workflow file.